src/api/handlers/settlements/{id}.js

```diff
@@ -33,6 +33,7 @@
 'use strict'
```

Comment:
Just to validate:

I believe the `HapiEventPlugin` Hapi plugin auto-magically handles the trace and the ingress audit.

However, I do not recall if it handles the "egress" audit event for the response going out?

@vgenev FYI.